humanoid_league_game_controller/scripts/sim_gamestate.py

```diff
@@ -2,6 +2,8 @@
 
 # This script was based on the teleop_twist_keyboard package
 # original code can be found at https://github.com/ros-teleop/teleop_twist_keyboard
+# The script provides a simple mechanism to test robot behaviour to different gamestates,
```

Comment:
for different gamestates?

Reply:
I think it should be "in different gamestates"